URBNSwiftAlert/Classes/URBNSwAlertStyler.swift

```diff
@@ -95,11 +95,6 @@ public struct URBNSwAlertStyler {
     }
     
     /**
-     * Insets of the alert from the top / left / bottom / right of the screen.
-     */
-    public var alertWrappingInsets: UIEdgeInsets?
```

Comment:
I think we do still want this, or bring back the `minAlertWidth` and `maxAlertWidth` properties